lib/pure/smtp.nim

```diff
@@ -28,20 +28,32 @@
 ## For SSL support this module relies on OpenSSL. If you want to
 ## enable SSL, compile with ``-d:ssl``.
 
-import net, strutils, strtabs, base64, os
+import net, strutils, strtabs, base64, os, mimetypes, sequtils
 import asyncnet, asyncdispatch
+import mersenne , times ## we need randomness for the boundary
```

Comment:
Please keep the style consistent, in this case there should be no space before the comma.